draft-ietf-rats-eat.md

```diff
@@ -288,54 +288,40 @@ An entity may have strong security defenses against hardware invasive attacks.
+* Claims are defined in CDDL and serialized using CBOR or JSON
+* Security envelopes based on COSE and JOSE
+* Nesting of claims sets and tokens to represent complex and compound devices
+* A profile mechanism for specifying and identifying specific token formats for specific use cases
 
-This specification extends those specifications by defining additional claims for attestation.
-This specification also describes the notion of a "eat_profile" that can narrow the definition of an EAT, ensure interoperability and fill in details for specific usage scenarios.
-This specification also adds some considerations for registration of future EAT-related claims.
+EAT uses the name/value pairs the same as CWT and JWT to identify individual claims.
+{{theclaims}} defines common attestation-oriented claims that are added to the CWT and JWT IANA registries.
+As with CWT and JWT, no claims are mandatory and claims not recognized should be ignored.
 
-The identification of a protocol element as an EAT, whether CBOR or JSON encoded, follows the general conventions used by CWT, JWT.
-Largely this depends on the protocol carrying the EAT.
-In some cases it may be by content type (e.g., MIME type).
-In other cases it may be through use of CBOR tags.
-There is no fixed mechanism across all use cases.
+Unlike, but compatible with CWT and JWT, EAT defines claims using Concise Data Definition Language (CDDL) {{RFC8610}}.
+In most cases the same CDDL definition is used for both the CBOR/CWT serialization and the JSON/JWT serialization.
 
-This specification adds one more top-level token type:
+Like CWT and JWT, EAT uses COSE and JOSE to provide authenticity, integrity and optionally confidentiality.
+EAT places no new restrictions on cryptographic algorithms, retaining all the cryptographic flexibility of CWT, COSE, JWT and JOSE.
 
-* Detached EAT Bundle, {{DEB}}
+EAT defines a means for nesting tokens and claims sets to accommodate composite devices that have multiple subsystems and multiple attesters.
+Full tokens with security envelopes may be embedded in an enclosing token.
+The nested token and the enclosing token do not have to use the same encoding (e.g., a CWT may be enclosed in a JWT).
 
-A detached EAT bundle is structure to hold a collection of detached claims sets and the EAT that separately provides integrity and authenticity protection for them.
-It can be either CBOR or JSON encoded.
+EAT adds the ability to detach claims sets and send them separately from a security enveloped EAT that contains a digest of the detached claims set.
 
-Last, the definition of other token types is allowed.
-Of particular use may be a token type that provides no authenticity or integrity protection at all for use with transports like TLS that do provide that.
+This document registers no media or content types for the identification of the type of EAT, its serialization format or security envelope.
+That is left for a follow-on document.
```

Comment:
Superfluous?

```suggestion
```

Reply:
I don't think it is superfluous because tags and media types are needed. They are committed for standards process reasons, not for technical reasons.